traveler-swift-core/TravelerKit/AuthPath.swift

```diff
@@ -37,6 +38,10 @@ enum AuthPath {
         var urlRequest = URLRequest(url: baseURL)
 
         switch self {
+        case .storeAttributes(let attributes, let travelerId):
+            urlComponents.path = "/v1/order/\(travelerId)"
```

Comment:
This is not the correct endpoint to store attributes. It should be /v1/traveler/{id}